components/org.wso2.carbon.identity.api.user.organization/org.wso2.carbon.identity.rest.api.user.organization.v1/src/main/resources/user.organization.yaml

```diff
@@ -163,6 +164,14 @@ components:
       schema:
         type: boolean
         default: false
+    applicationNameQueryParam:
+        in: query
+        name: applicationName
+        required: false
+        description:
+            Filter organizations that user has access due to roles of the particular application
```

Comment:
there is already a filter query param defined for this API. can't we reuse it?

Reply:
Agreed on using a separate param